sql/src/main/java/io/crate/protocols/postgres/PostgresWireProtocol.java

```diff
@@ -452,6 +469,14 @@ private void handleParseMessage(ByteBuf buffer, final Channel channel) {
         Messages.sendParseComplete(channel);
     }
 
+    private void handlePassword(ByteBuf buffer, final Channel channel) {
+        String passwd = readCString(buffer);
```

Comment:
would it make sense to read this as char array directly? (https://stackoverflow.com/questions/8881291/why-is-char-preferred-over-string-for-password )